composer.json

```diff
@@ -18,10 +20,11 @@
         "drupal/console": "^1.6",
         "drupal/config_devel": "~1.2",
         "drush/drush": "^9",
+        "drupal/devel": "^1.2",
         "drupal/drupal-extension": "^4.0.0@alpha",
         "openeuropa/code-review": "^0.2",
         "openeuropa/task-runner": "^0.5",
-        "openeuropa/oe_theme": "dev-master",
+        "openeuropa/oe_theme": "dev-OPENEUROPA-480",
```

Comment:
We would always commit the theme and remove the branch as part of the final commit process. I agree that now the platform is evolving and growing we should start using minimum version increases for components.

Reply:
Yep, agreed on not using `dev-master` anymore and tagging theme releases more often, especially after https://github.com/openeuropa/composer-artifacts/pull/1 will be merged.

Reply:
This should be taken care of before this is merged into the master branch, or this WILL break the master when the sister PR in OE Theme is merged and the corresponding branch deleted. If we don't address this then the `dev-OPENEUROPA-480` branch will not be resolved by Composer, and it will not be able to build the dependencies.

The best way to handle tickets that require dependent changes in multiple components is to make subtasks for each component, and handle them separately. This allows to push the dependent changes through first, then tag a new release and use this in the depending component.